### ISSUE TITLE
INSTALLED_APPS weird behavior

### ISSUE THREAD

**Issue by mzilinec** [botshot/botshot#34]
For some reason, these have different effect.
```INSTALLED_APPS = ('botshot', ...```
```INSTALLED_APPS = ('botshot.apps.BotshotConfig', ...```

For example, when running unit tests, the second variant crashes with "default.root not found".
This indicates that ready() may not be called in the first case.

**Comment by prihoda** [comment]
Yes the second one is needed to call the ready function. We might not need the ready function in the future but for now we have to use botshot.apps.BotshotConfig

**Comment by mzilinec** [comment]
Well at least we can use the first one for testing.
